packages/docs/users/roles.yaml

```diff
@@ -18,53 +18,52 @@ _ref:
     pageId: roles
     pageTitle: Roles
     section: User Authentication
-    filePath: concepts/roles.yaml
+    filePath: users/roles.yaml
     content:
       - id: introduction
         type: MarkdownWithCode
         properties:
-          content: |
-            Roles can be used to limit user access to certain pages. Only users with the role will be able to see the page, and the page will be filtered from menus if the user does not have the role.
+          content:
+            _nunjucks:
+              on:
+                version:
+                  _ref: version.yaml
+              template: |
+                Roles can be used to limit user access to certain pages. Only users with the role will be able to see the page, and the page will be filtered from menus if the user does not have the role.
```

Comment:
Sentence can be a bit confusing. What 'role' are we referring to? 

Maybe something like 'Only users with the role linked to the page will be able to see that page...'